plugins/jaeger/src/components/panel/TracesChart.tsx

```diff
@@ -102,6 +102,15 @@ const TracesChart: React.FunctionComponent<ITracesChartProps> = ({ name, traces,
                 showDetails(<Trace name={name} trace={node.data.trace} close={(): void => showDetails(undefined)} />);
               }
             }}
+            // @ts-ignore as the typing expects NodeProps but actually has Node
+            renderNode={(ctx: CanvasRenderingContext2D, props: Node): void => {
+              const hasError = isIDatum(props.data) ? doesTraceContainsError(props.data.trace) : false;
+
+              ctx.beginPath();
+              ctx.arc(props.x, props.y, props.size / 2, 0, 2 * Math.PI);
+              ctx.fillStyle = hasError ? '#FF0000' : props.style.color;
```

Comment:
Can you use the error color defined by Patternfly please:

```suggestion
              ctx.fillStyle = hasError ? '##c9190b' : props.style.color;
```